external/fv3fit/fv3fit/reservoir/config.py

```diff
@@ -58,20 +58,6 @@ class BatchLinearRegressorHyperparameters:
     use_least_squares_solve: bool = False
 
 
-@dataclass
-class ReadoutHyperparameters:
```

Comment:
This class is no longer needed, as the readout is not responsible for creating and training a regressor.